src/libraries/System.IO.FileSystem.Watcher/src/System.IO.FileSystem.Watcher.csproj

```diff
@@ -109,9 +109,6 @@
     <Compile Include="$(CommonPath)System\Text\ValueUtf8Converter.cs"
              Link="Common\System\Text\ValueUtf8Converter.cs" />
   </ItemGroup>
-  <ItemGroup Condition="'$(TargetPlatformIdentifier)' == 'UnknownUnix'">
-    <Compile Include="System\IO\FileSystemWatcher.UnknownUnix.cs" />
-  </ItemGroup>
```

Comment:
We'd had this here to help with bring-up for additional platforms.  I assume it's being deleted because UnknownUnix is no longer ever being set?

Reply:
Correct but there is no tfm that includes it and there is currently no concept of an unknown Unix RID in our tfm matrix.